.github/scripts/wait_for_looker.py

```diff
@@ -0,0 +1,34 @@
+import ssl
+import sys
+import time
+import urllib.request
+
+# turn off ssl checking
+ctx = ssl.create_default_context()
+ctx.check_hostname = False
+ctx.verify_mode = ssl.CERT_NONE
+
+MAX_RETRIES = 160
+ATTEMPTS = 1
+while MAX_RETRIES:
+    retry_msg = f"after {ATTEMPTS} attempts: {MAX_RETRIES} retries remaining."
+    try:
+        status = urllib.request.urlopen(
+            "https://localhost:10000/login", context=ctx
```

Comment:
Is this just to see if the looker server is alive?
```suggestion
            "https://localhost:10000/alive", context=ctx
```

Reply:
I'll play with that.